docs/maya/skin_merge.md

```diff
@@ -0,0 +1,66 @@
+# AdnSkinMerge
+
+AdnSkinMerge is a Maya deformer to merge animation geometries with the results provided by AdnSkin. This deformer allows for the merging of several animation and simulation meshes into a single final mesh.
+
+The influence simulation or animation meshes will have on the final mesh can be freely painted and modified by painting a blend weights map.
+
+### How To Use
+
+AdnSkinMerge makes use of its own User Interface to create and modify the deformer.
+
+To create an AdnSkinMerge deformer within a Maya scene, the following inputs must be provided:
+
+  - **Final Mesh (F)**: Mesh to apply the merge results.
+  - **Animation Mesh List (Al)**: Mesh(es) to drive the simulation skin.
+  - **Simulation Mesh List (Sl)**: Mesh(es) with either an AdnSkin deformer applied or with the results from the skin simulation applied.
+
+The process to create an AdnSkinMerge deformer is:
+
+1. Press ![Skin merge button](images/adn_skin_merge.png){style="width:4%"} in the AdonisFX shelf or *Skin Merge* in the AdonisFX menu, under the *Deformers* submenu in the *Create* section.
+2. The following UI will get displayed. From here the required meshes for the deformer will be selected and added. At least one mesh must be added in each field to be able to create the deformer. A custom name and initialization time can also be set in this UI.
+    - To add meshes to any list, select the meshes in the scene and click the respecitve *Add Selected* button.
+    - If you wish to remove a single element from the list, select it in the Skin Merge UI and press the *Remove Selected* button.
+    - You may also clear any list fully by pressing the respective *Clear* button.
```

Comment:
We should add that duplicated meshes is not allowed, as well as the same mesh in both anim and sim list is not allowed either

Reply:
Same mesh in both lists is allowed but not recommended, we display a warning message, but it is allowed.
I have added two additional points detailing this.